From: Colin Ian King <colin.king@canonical.com>
Date: Sat, 5 Aug 2017 14:11:50 +0100
Subject: IB/hfi1: fix spelling mistake: "Maximim" -> "Maximum"
Patch-mainline: v4.14-rc1
Git-commit: 028e0a67ea679ff958176eac97d1319a8d685fad
References: bsc#1060463 FATE#323043

Trivial fix to spelling mistake in pr_warn_ratelimited warning message

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Acked-by: Dennis Dalessandro <dennis.dalessandro@intel.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/hfi1/mad.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/infiniband/hw/hfi1/mad.c
+++ b/drivers/infiniband/hw/hfi1/mad.c
@@ -173,7 +173,7 @@ static struct trap_node *check_and_add_t
 			trap_list->list_len++;
 			list_add_tail(&trap->list, &trap_list->list);
 		} else {
-			pr_warn_ratelimited("hfi1: Maximim trap limit reached for 0x%0x traps\n",
+			pr_warn_ratelimited("hfi1: Maximum trap limit reached for 0x%0x traps\n",
 					    trap->data.generic_type);
 			kfree(trap);
 		}
